tests/tutorials/test_tutorials.py

```diff
@@ -124,6 +124,9 @@ def test_nlp_cnn():
 def test_onnx_super_resolution():
     assert _test_tutorial_nb('onnx/super_resolution')
 
+def test_onnx_export_mxnet_to_onnx():
+    assert _test_tutorial_nb('onnx/export_mxnet_to_onnx')
```

Comment:
these tests are currently disabled from running on the nightly test run.

Reply:
Fails CI build if new test not added in test_tutorials. If this is currently disabled, do we need it as a part of CI lint check? @marcoabreu

Reply:
yes